pkg/loader/openapi.go

```diff
@@ -373,6 +373,147 @@ func parseServer(server *openapi3.Server) (string, error) {
 	return s, nil
 }
 
+// openAPI3SchemaToHumaV2Schema converts an openapi3.Schema to a humav2.Schema
+func openAPI3SchemaToHumaV2Schema(schema *openapi3.Schema) *humav2.Schema {
```

Comment:
Junie did this for me. I checked it out and it seems good, but would appreciate some eyes on this.

Reply:
![image](https://github.com/user-attachments/assets/a7d6a606-7418-42d0-94a8-568463199e79)